server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -165,11 +165,15 @@ private UserDto createDto(DbSession dbSession, NewUser newUser) {
       setEncryptedPassword(password, userDto);
     }
 
-    List<String> scmAccounts = sanitizeScmAccounts(newUser.scmAccounts());
+    List<String> scmAccounts = sanitizeStringList(newUser.scmAccounts());
     if (scmAccounts != null && !scmAccounts.isEmpty() && validateScmAccounts(dbSession, scmAccounts, login, email, null, messages)) {
       userDto.setScmAccounts(scmAccounts);
     }
 
+    // TODO should we validate that one of the secondary email is already by another user (as email, scm accout or secondary email)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjl%2FSONAR-10248%2Fsecondary_emails_sync&issues=AWDHC7TeJC-F6PRGRIOj&open=AWDHC7TeJC-F6PRGRIOj)

[ISSUE_KEY]: AWDHC7TeJC-F6PRGRIOj